src/cities/cityConfigs.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+import CityConfig from './CityConfig'
+
+const configs = [
+  new CityConfig('neuburgschrobenhausenwohnraum', 'neuburg.wohnen.integreat-app.de', true, 'Neuburg-Schrobenhausen', null, 'https://www.neuburg-schrobenhausen.de/Impressum.n10.html'),
```

Comment:
The offical url is https://raumfrei.neuburg-schrobenhausen.de/

Reply:
Changed it :)